library/result.lisp

```diff
@@ -52,6 +54,14 @@
       ((Ok x) x)
       ((Err x) x)))
 
+  (declare unwrap-or-error (Signalable :a => (Result :a :b) -> :b))
```

Comment:
I think this should be defined on the `Unwrappable` class.

Reply:
I'll keep thinking about this, but my initial thought is that this needs to be just for `Result` because other `Unwrappable` instances (currently only `Optional`) don't necessarily carry an error/fail type to be declared `Signalable`. 

Using `unwrap` on an `Optional` already provides a default error string, which is all that an ideal `unwrap-or-error` would provide anyways. 

In order to avoid confusion it could be renamed something like `ok-or-error`.

Reply:
I fixed the package definitions and changed to `ok-or-error`.